ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1864,3 +1864,24 @@ def test_paddle_diff(
         prepend=prepend[0],
         append=append[0],
     )
+
+
```

Comment:
You didn't generate any input strategies for the test using the `@handle_frontend_test` decorator. Kindly add these and run the test using `pytest` to see if it passes with all backends or not

Reply:
Thank you, I did not really understand how the testing worked indeed. In my latest commit I have added a `@handle_frontend_test` but my tests are not working. 
For example with input data `[0]` and value 1 I get the error that my solution is 1 while the groundtruth solution is 2.
After inspecting the testing method, I think that this is caused because the same array is used to test my implementation of the frontend function first and then the ground truth version later. The problem is that a side effect of the padddle.increment value is that the data that you give it is incremented itself. So when the frontend is called with x=`[0]` and value=1, the correct value of 1 is returned, but causes `x=[1]` which is why when the groundtruth value is calculated afterwards it adds 1 to 1 and says that the 1 != 2. I am a bit lost in how to properly test this method, so hope you can help me!

Reply:
@FelixHuyghe Try copying the array using something like `ivy.copy_array` before passing it to `ivy.inplace_increment`. See if that works for you and let me know :)

Reply:
Yes that works! I copied the array before passing it to ivy.inplace_increment and now my tests pass! However, I think this implementation does not fully reflect the original paddle one, because if you pass an array to the paddle function, a side effect is that array is incremented "in place" and with my frontend implementation when the function returns the provided array is still the same.

Reply:
> Yes that works! I copied the array before passing it to ivy.inplace_increment and now my tests pass (when x is of shape [1] anyways)! However, I think this implementation does not fully reflect the original paddle one, because if you pass an array to the paddle function, a side effect is that array is incremented "in place" and with my frontend implementation when the function returns the provided array is still the same (because of the copy).

I see what you mean. I think the test for `ivy.inplace_increment` should give you an idea of how to test this frontend function because it is kind of similar to what's happening with `ivy.inplace_increment` I would think so.